charts/ccsm-helm/charts/tasklist/templates/deployment.yaml

```diff
@@ -25,12 +25,7 @@ spec:
           - name: GRAPHQL_PLAYGROUND_SETTINGS_REQUEST_CREDENTIALS
             value: {{ default "include" .Values.graphqlPlaygroundRequestCredentials | quote }}      
         resources:
-          requests:
-            cpu: 500m
-            memory: 512Mi
-          limits:
-            cpu: 1000m
-            memory: 768Mi
+          {{- toYaml .Values.resources | nindent 10 }}
```

Comment:
:question: `.Values.resources` here refers to all values under `tasklist` in the parent chart? Is that then equivalent to `.Values.global.tasklist.resources`?

Reply:
No it is not equivalent, since it was not defined under global. Global is a special variable which is really accessible everywhere (in all subcharts)

Defining something like:

```
tasklist:
  resource:
```
Would overwrite the resources in the tasklist chart and is only available there.